regparser/diff/tree.py

```diff
@@ -9,7 +9,7 @@
 MODIFIED = 'modified'
 DELETED = 'deleted'
 
-_whitespace = re.compile(u'\s+', re.UNICODE)    # aware of "thin" spaces, etc.
+_whitespace = re.compile(u'\s', re.UNICODE)    # aware of "thin" spaces, etc.
```

Comment:
Are there cases where multiple spaces are relevant? The GPO guidelines insist on one space after periods:

> 2.49. A single justified word space will be used between sentences. 
> This applies to all types of composition. 

There are other use cases, but we don't really support them as far as I can tell (using multiple spaces for indentation, for example).

It's less clear whether we should be collapsing multiple line breaks, however. What I'd  prefer here is `u'\h+'`, but Python's `re` module doesn't support `\h`, so implementing this would have to be a two-step process if we chose to do that. Not a reason not to merge, just a note.